harmonica/gravity_corrections.py

```diff
@@ -69,3 +74,72 @@ def bouguer_correction(topography, density_crust=2670, density_water=1040):
     density[oceans] = -1 * (density_water - density_crust)
     bouguer = 1e5 * 2 * np.pi * GRAVITATIONAL_CONST * density * topography
     return bouguer
+
+def atmospheric_correction(topography):
+    """
+    Calculates atmospheric correction as per Hinze 2005 eqn 5.
+    
+    .. math::
+
+        g_{atm} = 0.874 - 9.9E-5 h + 3.56E-9 h^2 
+
+    in which :math:`h` is the elevation of the station above the ellipsoid
+    and :math:`g_{atm}` is the gravitational effect of the atmospheric the air
+    mass in mGal.
+
+    Parameters
+    ----------
+    topography : array or :class:`xarray.DataArray`
+        Topography height and bathymetry depth in meters.
+        Should be referenced to the ellipsoid (ie, geometric heights).
+        
+
+    Returns
+    -------
+    atmos_correction : array or :class:`xarray.DataArray`
+    The gravitational effect of the atmosphere in mGal
+
+    """
+    atmos_correction = 0.874 - 9.9e-05 * topography + 3.56e-09 * topography**2
+    
+    return atmos_correction
+
+
+def spherical_bouguer_cap_correction(topography):
+    """
+    Calculates spherical cap correction for the Bouguer slab as per Hinze 2013.
```

Comment:
Is this a correction applied to the Bouguer correction or the effect of a spherical cap? In other words, does this require a Bouguer correction first or can be it directly applied to the gravity disturbance?

Reply:
This is applied in addtionto the bouguer correction, but there is actually a better formulation of the bouguer correction (spherical bouguer correction) that takes the curvature into account.  Perhaps we should use that instead of the bouguer correction + spherical cap correction?

The spherical bouguer correction is 2pi * G * density (mu * h - lamda -R) (laFeher 1991 Geophysics, 56) where mu and lamda are dimensionless constants (formulas given in that paper to calculate).  

Using the spherical bouguer correction is recommended in the "New standards for reducing gravity data) by Hinze 2005 (Geophysics 70) so we should probably adopt that

Reply:
It's my understanding that the spherical cap correction is applied to the Bouguer correction.  The Bouguer correction has 3 parts, the infinite slab (2piGph, also known as Bullard A), the spherical cap correction (also known as Bullard B) and the terrain correction (Bullard C).